gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/osUtils.kt

```diff
@@ -36,6 +36,7 @@ internal val currentArch by lazy {
     when (osArch) {
         "x86_64", "amd64" -> Arch.X64
         "aarch64" -> Arch.Arm64
+        "x86" -> error("32-bit arch not supported: $osArch")
```

Comment:
Not sure if x86 any different than anything else here, maybe just change "Unknown" -> "Unsupported" in branch below.

Reply:
done